/home/denhart.c/.local/lib/python3.7/site-packages/psaw/PushshiftAPI.py:153: UserWarning: Unable to connect to pushshift.io. Retrying after backoff.
  warnings.warn("Unable to connect to pushshift.io. Retrying after backoff.")
Extracting posts from praw .....
175909 posts from Reddit retrieved.
Serializing data .....
Writing data to mongodb .....
Write response:

175909 documents inserted into mongo.

Program completed.
